### DISCUSSION TITLE
Release Artifact 0.71.0

### DISCUSSION THREAD

**Discussion by Arvindh-Mojo** [open-telemetry/opentelemetry-collector-contrib#18475] [General]
Looks like the release `0.71.0` did not produce the required artifact. 
https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/4128527733

I want to try the `0.71.0` but seems like we are having some build issues with it. Can you confirm or am i missing something ?

**Comment by Arvindh-Mojo** [comment]
@atoulme is this something you could help ? LMK.. Thanks !!

**Reply by atoulme** [reply]
We should be good now. Look here: https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.71.0

Please report any and all issues you find, thanks!
